src/sidebar/store/modules/selection.js

```diff
@@ -97,7 +120,28 @@ function init(settings) {
 }
 
 const update = {
-  CLEAR_SELECTION: function() {
+  CLEAR_SELECTION: function(state) {
+    let selectedTab = state.selectedTab;
+    if (selectedTab === uiConstants.TAB_ORPHANS) {
```

Comment:
There was a !selectedTab || selectedTab === uiConstants.TAB_ORPHANS in here before but I don't see how selectedTab could be falsey.